services/src/unit-test/java/uk/gov/moj/sdt/services/messaging/IndividualRequestMdbTest.java

```diff
@@ -73,76 +56,80 @@ public void setUp() {
+        final ObjectMessage objMessage = mock(ObjectMessage.class);
+        when(objMessage.getObject()).thenReturn((Serializable) sdtMessage);
 
         mockTargetSubmissionService.processRequestToSubmit(sdtMessage.getSdtRequestReference());
 
-        EasyMock.expectLastCall();
-
-        EasyMock.replay(mockTargetSubmissionService);
+        verify(mockTargetSubmissionService).processRequestToSubmit(sdtMessage.getSdtRequestReference());
 
         individualRequestMdb.readMessage(objMessage);
 
-        EasyMock.verify(mockTargetSubmissionService);
-        EasyMock.verify(objMessage);
-
-        Assert.assertTrue("Test successfully completed", true);
-
+        assertTrue(true,TEST_SUCCESSFULLY_COMPLETED);
     }
 
     /**
      * Test method to test the scenario where the service throws a Data Access Exception.
      *
      * @throws JMSException if any during the test setup
      */
+
     @Test
-    public void readMessageDataAccessException() throws JMSException {
+    public void readMessageDataAccessExceptionTest() throws JMSException {
         individualRequestMdb = new IndividualRequestMdb(mockTargetSubmissionService);
 
-        // Create the actual message to send.
+        // Create the actual message to send
         final ISdtMessage sdtMessage = new SdtMessage();
         sdtMessage.setSdtRequestReference("ReadTest1");
 
         // Wrap the message in the ObjectMessage
-        final ObjectMessage objMessage = EasyMock.createMock(ObjectMessage.class);
-        objMessage.setObject((Serializable) sdtMessage);
-        EasyMock.expectLastCall().anyTimes();
-
-        EasyMock.expect(objMessage.getObject()).andReturn((Serializable) sdtMessage);
-        EasyMock.replay(objMessage);
+        final ObjectMessage objMessage = mock(ObjectMessage.class);
+        when(objMessage.getObject()).thenReturn((Serializable) sdtMessage);
 
-        mockTargetSubmissionService.processRequestToSubmit(sdtMessage.getSdtRequestReference());
-
-        EasyMock.expectLastCall().andThrow(new RuntimeException("Data error occurred"));
-
-        EasyMock.replay(mockTargetSubmissionService);
+        doThrow(new RuntimeException("Data error occurred"))
+            .when(mockTargetSubmissionService).processRequestToSubmit(sdtMessage.getSdtRequestReference());
 
         try {
             individualRequestMdb.readMessage(objMessage);
-            Assert.fail("RuntimeException was expected to be thrown");
+            fail("RuntimeException was expected to be thrown");
         } catch (final RuntimeException re) {
-            Assert.assertTrue("Expected to throw the exception", true);
+            assertTrue(true,"Expected to throw the exception");
         }
 
-        EasyMock.verify(mockTargetSubmissionService);
-        EasyMock.verify(objMessage);
+        Mockito.verify(mockTargetSubmissionService).processRequestToSubmit(sdtMessage.getSdtRequestReference());
+        Mockito.verify(objMessage).getObject();
```

Comment:
Can use an import static for the Mockito verify method (i.e. import static org.mockito.Mockito.verify) then you won't need to prefix it with Mockito here.

Reply:
done